projects/ion/src/lib/alert/alert.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, Input } from '@angular/core';
+
+export type AlertType = 'success' | 'info' | 'alert' | 'danger';
+
+export interface IonAlertProps {
+  message: string;
+  type?: AlertType;
+  closable?: boolean;
+  width?: number;
+  height?: number;
+}
+
+@Component({
+  selector: 'ion-alert',
+  templateUrl: './alert.component.html',
+  styleUrls: ['./alert.component.scss'],
+})
+export class AlertComponent {
+  @Input() message!: string;
+  @Input() type?: AlertType = 'success';
+  @Input() closable? = false;
+  @Input() width? = 235;
+  @Input() height? = 30;
```

Comment:
You can remove this sizes, this element need adapt with the screen, keeping 100% of where is inserted.